src/components/loadable-item.cjsx

```diff
@@ -31,15 +31,16 @@ module.exports = React.createClass
   componentDidUpdate: (oldProps) -> @reload(oldProps)
 
   reload: (oldProps) ->
-    {id, store, load, actions, options} = @props
+    {id, store, load, actions, options, isLoading} = @props
     return unless id?
 
     # Skip reloading if all the props are the same (the case in the Calendar for some reason)
     return if oldProps.id is id and oldProps.store is store and oldProps.actions is actions and
       oldProps.load is load and _.isEqual(oldProps.options, options)
 
     load ?= actions.load
-    unless store.isNew(id, options)
+    isLoading ?= store.isLoading
+    unless store.isNew(id, options) and not isLoading?(id, options)
```

Comment:
Yeah... so I tried that and it didn't work out well.  there's code in the task plans that check if the stores are reading & homework specs that expect the store to load unless it's new.  My moving the isLoading condition first the isNew status isn't checked.  

I could fixup the specs but am worried that there might be other untested areas that have the same logic.  Probably best to leave as is despite the better readability of your version